core/arch/arm/plat-nuvoton/conf.mk

```diff
@@ -0,0 +1,39 @@
+PLATFORM_FLAVOR ?= npcm845x
+
+ifeq ($(PLATFORM_FLAVOR),npcm845x)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+CFG_ARM64_core ?= y
+endif #npcm845x
+
+CFG_USER_TA_TARGETS ?= ta_arm64
+
+ifeq ($(platform-debugger-arm),1)
```

Comment:
Please remove these unused lines too.

Reply:
CFG_USER_TA_TARGETS ?= ta_arm64 is used, otherwise it requires CROSS_COMPILE for 32 bits as well